EIPS/eip-1.md

```diff
@@ -229,7 +229,7 @@ If the EIP isn't ready, the editor will send it back to the author for revision,
 
 Once the EIP is ready for the repository, the EIP editor will:
 
-- Assign an EIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in [the Issues section of this repository](https://github.com/ethereum/EIPs/issues) about this EIP)
+- Assign an EIP number (generally the PR number)
```

Comment:
```suggestion
- Assign an EIP number (generally the PR number, but the decision is with the editors)
```

Reply:
I would also like to see the change proposed here by Alex, but if you are against it @poojaranjan we can merge this as is (it is an improvement even without Alex's suggestion) and then debate the addition of Alex's change to a separate PR.

Reply:
I am good!